frontend/debian/cuttlefish-orchestration.postinst

```diff
@@ -11,6 +11,11 @@ case "$1" in
         # The cvdnetwork group is created by cuttlefish-base
         usermod -a -G cvdnetwork,kvm _cvd-executor
     fi
+
+    # Reload nginx with the orchestration configuration
+    rm /etc/nginx/nginx.conf
+    ln -s /etc/cuttlefish-orchestration/nginx.conf /etc/nginx/nginx.conf
+    sudo nginx -s reload
```

Comment:
Could you put this configuration file in `/etc/nginx/conf.d/host_orchestrator.conf` instead? That avoids having to override the entire nginx config and plays well with other services that could be installed in the host.
That would also allow you to enable and disable that configuration on host_orchestrator service start/stop, simply by changing the file extension and reloading nginx.

Reply:
> different nginx packages can potentially install different defaults we won't be having control over

This would still be problem if the other package is installed **after** `cuttlefish-orchestration`, as that package would still override our config.

I'm thinking other users of the orchestration package could decide to install something like a /health endpoint or some other service in the host for which they want to use nginx but we're taking over nginx config completely.

> one server and different location entries

I don't know what this means, I'm not familiar enough with nginx configuration. Based on my previous argument, I would still prefer the conf.d approach.

Reply:
Agree, added our own specific config under `/etc/nginx/conf.d`.